ui/helpers/utils/i18n-helper.tsx

```diff
@@ -110,19 +115,21 @@ export async function fetchLocale(localeCode) {
 
 const relativeTimeFormatLocaleData = new Set();
 
-export async function loadRelativeTimeFormatLocaleData(localeCode) {
+export async function loadRelativeTimeFormatLocaleData(
+  localeCode: string,
+): Promise<any> {
   const languageTag = localeCode.split('_')[0];
   if (
     Intl.RelativeTimeFormat &&
-    typeof Intl.RelativeTimeFormat.__addLocaleData === 'function' &&
+    typeof (Intl.RelativeTimeFormat as any).__addLocaleData === 'function' &&
     !relativeTimeFormatLocaleData.has(languageTag)
   ) {
     const localeData = await fetchRelativeTimeFormatData(languageTag);
-    Intl.RelativeTimeFormat.__addLocaleData(localeData);
+    (Intl.RelativeTimeFormat as any).__addLocaleData(localeData);
   }
 }
 
-async function fetchRelativeTimeFormatData(languageTag) {
+async function fetchRelativeTimeFormatData(languageTag: string): Promise<any> {
```

Comment:
Would normally be an okay usage of Any, but there is a 'Json' type exported from '@metamask/utils' we should use for this.

Reply:
Good to know !! changed 🙏